domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParser.java

```diff
@@ -41,17 +54,17 @@ private boolean fieldContainsEmpties(String field) {
         return field.contains(" ") || field.contains("..");
     }
 
-    private Optional<ShowCondition> buildShowCondition(String[] andConditions) {
+    private Optional<ShowCondition> buildShowCondition(String[] andConditions, String andOrOperator) {
```

Comment:
can we call `andOrOperator` conditionalOperator? since it gets assigned both the AND and the OR operator

Reply:
DONE